### ISSUE TITLE
Errors compiling with kernel 4.10.2

### ISSUE THREAD

**Issue by storcium** [elmystico/EnhanceIO#7]
Hi elmystico, hopefully you are still interested in this project.  Trying to compile under kernel 4.10 fails with many errors like this. I read that driverfs_dev was removed in 4.8:

eio_conf.c: In function ‘eio_init_srcdev_props’:
/usr/local/ISS/EnhanceIO/Driver/enhanceio/eio_conf.c:1471:34: error: ‘struct gendisk’ has no member named ‘driverfs_dev’
      dmc->disk_dev->bdev->bd_disk->driverfs_dev) {

**Comment by elmystico** [comment]
Yes, I'm stuck with kernel 4.7 because of some changes in kernel code.
I'm quite busy doing other stuff...
But now when you mention, I can see there are some patches flying around up to 4.10 kernel, I'll try to merge them into my codebase and see how it goes.
I'll report back but it will take a week or so.

o/
